src/libostree/ostree-repo-pull.c

```diff
@@ -1241,13 +1241,28 @@ meta_fetch_on_complete (GObject           *object,
                                FALSE, &metadata, error))
         goto out;
 
-      /* For commit objects, check the GPG signature before writing to the repo,
-       * and also write the .commitpartial to say that we're still processing
-       * this commit.
+      /* For commit objects, compute the hash and check the GPG signature before
+       * writing to the repo, and also write the .commitpartial to say that
+       * we're still processing this commit.
        */
       if (objtype == OSTREE_OBJECT_TYPE_COMMIT)
         {
           GVariant *detached_data = g_hash_table_lookup (pull_data->fetched_detached_metadata, checksum);
```

Comment:
Can we add a `/* NULL if it wasn't found */` somewhere here to clarify that `detached_data` can be `NULL`?